src/Table/Column/Flag.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Ui\Table\Column;
+
+use Atk4\Data\Model;
+use Atk4\Ui\Table;
+
+/**
+ * Column for formatting country code as flags.
+ */
+class Flag extends Table\Column
+{
+    /** @var string Name of model field which contains country ALPHA-2 (2 letter) codes. */
+    public $code_field;
```

Comment:
Didn't we switch to camelCase property names?

Reply:
yes, we did everywhere!